tests/translator/input/cloudwatchevent_intrinsics.yaml

```diff
@@ -0,0 +1,32 @@
+Parameters:
+  PathA:
+    Type: String
+    Default: "SomeInputPath"
+  PathB:
+    Type: String
+    Default: "AnotherInputPath"
+
+Conditions:
+  PathCondition:
+    Fn::Equals:
+      - true
+      - true
+
+Resources:
+  LambdaFunction:
+    Type: 'AWS::Serverless::Function'
+    Properties:
+      CodeUri: s3://sam-demo-bucket/hello.zip
+      Handler: hello.handler
+      Runtime: python2.7
+      Events:
+        OnTerminate:
+          Type: CloudWatchEvent
+          Properties:
+            EventBusName: !Join [ "", [ "Event", "Bus", "Name" ] ]
+            Pattern:
+              detail:
+                state:
+                  - terminated
```

Comment:
Q: are intrinsic functions not supported for this field?

Reply:
Added an intrinsic here.